utils/ecs.py

```diff
@@ -58,7 +57,9 @@ def get_acs_connection_info(module):
             module.fail_json(msg="access key is required")
 
     if not secret_key:
-        if 'ACS_SECRET_ACCESS_KEY' in os.environ:
+        if 'ALICLOUD_SECRET_ACCESS_KEY' in os.environ:

```

Comment:
Please change 'ALICLOUD_SECRET_ACCESS_KEY' to 'ALICLOUD_SECRET_KEY' in the line 68.